reference-implementation/test/writable-stream.js

```diff
@@ -473,154 +473,6 @@ test('If close is called on a WritableStream in waiting state, ready will be ful
   }, 0);
 });
 
-test('If sink rejects on a WritableStream in writable state, ready will return a fulfilled promise', t => {
```

Comment:
you've dropped this since the "returning a rejected promise (second write)" test would cover most of this?